addons/website/static/src/js/website.contentMenu.js

```diff
@@ -236,6 +238,7 @@ var EditMenuDialog = widget.Dialog.extend({
     add_menu: function () {
         var self = this;
         var dialog = new MenuEntryDialog(this, {menu_link_options: true}, undefined, {});
+        dialog.$modal.addClass('o_contentMenu_link_modal');
```

Comment:
Actually, if we decide to do it, it will also be a backend behavior (and all backend behaviors should be ok in frontend after websitepocalypse). I created this: https://www.odoo.com/web#id=34648&view_type=form&model=project.task&action=327&menu_id=4720